dts/arm64/qemu/qemu-virt-a53.dtsi

```diff
@@ -91,5 +100,36 @@
 			 */
 			reg = <0x0 0x0 0x0 DT_SIZE_M(64) /* 0x4000000 DT_SIZE_M(64) */>;
 		};
+
+		pcie: pcie@4010000000 {
+			compatible = "pci-host-ecam-generic";
+			device_type = "pci";
+			reg = <0x40 0x10000000 0x00 0x10000000>;
```

Comment:
Can you update the comment at the beginning of this file with the `dumpdtb` command you used for this?

Reply:
Hello Carlo! thank you for the review.
Actually, the same command in the current comment (i.e., `qemu-system-aarch64 -machine virt -cpu cortex-a53 -nographic -machine dumpdtb=virt.dtb`) already produces the pcie dtb information. So i do not think it needs to be changed.

I'm using host-tools from zephyr sdk 15.1

Reply:
perfect, thanks.